src/library-authoring/units/LibraryUnitPage.tsx

```diff
@@ -64,6 +65,7 @@ const HeaderActions = () => {
         iconBefore={Add}
         variant="primary rounded-0"
         disabled={readOnly}
+        onClick={openAddContentSidebar}
       >
         {intl.formatMessage(messages.newContentButton)}
       </Button>
```

Comment:
From the [ticket](https://github.com/openedx/frontend-app-authoring/issues/1619), it looks like we need to show an additional "Add New Component" button under the list of blocks in a unit (in addition to the "Add Content" button at the top).